packages/contextual-help/src/ContextualHelp.ts

```diff
@@ -0,0 +1,205 @@
+        }
+    }
+
+    private renderPopover(event: SlottableRequestEvent): void {
+        event.stopPropagation();
+
+        if (event.data === removeSlottableRequest) {
+            render(undefined, event.target as HTMLElement);
+            return;
+        }
+
+        import('@spectrum-web-components/popover/sp-popover.js');
+
+        const template = html`
+            <sp-popover class="popover">
+                <section>
+                    ${this.headline &&
+                    html`
+                        <h2 class="heading">${this.headline}</h2>
+                    `}
+                    <slot></slot>
+                    <div class="link">
+                        <slot name="link"></slot>
+                    </div>
+                </section>
+            </sp-popover>
+        `;
+
+        render(template, event.target as HTMLElement);
+    }
+
+    private renderDialog(event: SlottableRequestEvent): void {
+        event.stopPropagation();
+
+        if (event.data === removeSlottableRequest) {
+            render(undefined, event.target as HTMLElement);
+            return;
+        }
+
+        import('@spectrum-web-components/dialog/sp-dialog-wrapper.js');
+        const headlineVisibility = !this.headline ? 'none' : undefined;
+
+        const template = html`
+            <sp-dialog-wrapper
+                dismissable
+                underlay
+                headline=${ifDefined(this.headline)}
+                headline-visibility=${ifDefined(headlineVisibility)}
+            >
+                <slot></slot>
+                <div class="link">
+                    <slot name="link"></slot>
+                </div>
+            </sp-dialog-wrapper>
+        `;
+
+        render(template, event.target as HTMLElement);
+    }
+
+    protected override render(): TemplateResult {
+        const actualPlacement = this.isMobile.matches
+            ? undefined
+            : this.placement;
+
+        return html`
+            <sp-action-button
+                quiet
+                size="s"
+                id="trigger"
+                aria-label=${this.buttonAriaLabel}
+                ?is-active=${this.overlayOpen}
+            >
+                ${this.variant === 'help'
+                    ? html`
+                          <sp-icon-help-outline
+                              slot="icon"
+                          ></sp-icon-help-outline>
+                      `
+                    : html`
+                          <sp-icon-info-outline
+                              slot="icon"
+                          ></sp-icon-info-outline>
+                      `}
+            </sp-action-button>
+            <sp-overlay
+                trigger="trigger@click"
+                placement=${ifDefined(actualPlacement)}
+                type=${this.isMobile.matches ? 'modal' : 'auto'}
+                receives-focus="true"
+                .offset=${this.offset}
+                @sp-opened=${() => {
```

Comment:
`sp-opened` technically _late_ for this. The `slottable-request` timing is likely more appropriate for knowing when the Overlay is "open".